ethdb/src/main.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Parity Ethereum command-line tool for database interaction.
+
+#![warn(missing_docs)]
+
+extern crate common_types;
+extern crate serde;
+extern crate docopt;
+
+use common_types::{BlockNumber};
+
+use docopt::Docopt;
+use serde::Deserialize;
+
+const USAGE: &'static str = "
+Parity Ethereum interaction with the DB on-demand.
```

Comment:
I'd move this to a separate file and `include_str!` here if we are going to stick with docopt

Reply:
i've addressed this in commit ee203a618471d34514140fa6188ddbf147d3635c